src/documents/themes/yellow-swan/css/base.less

```diff
@@ -121,7 +121,7 @@ nav {
   letter-spacing:-.03em;
 }
 .speakers-name{font-size:1.6em;}
-.speakers-item p {font-size:1.4em;}
+.speakers-item p {font-size:1.6em;}
```

Comment:
Do not change default/base styles. Create a new sheet.

This way we can grab updates from https://github.com/braziljs/conf-boilerplate without worrying about conflicts.